### ISSUE TITLE
In practice, how do I deploy updates to my production server?

### ISSUE THREAD

**Issue by skosch** [thereactivestack/kickstart#17]
The answer to this may well be covered elsewhere, but since webpack + Meteor are still a somewhat unusual combination, I figured I'd ask here.

**tl;dr:** Can I somehow use `mup` to push to production whatever bundles webpack has generated?

Before webpack, I'd develop on my laptop and use arunoda's wonderful `mup` script to push new features to the VPS hosting the live app. For those unaware, `mup` sets up a standalone mongodb on the server and runs the Meteor node process via `forever`, which means I can push changes without ever having to restart anything or having to worry about downtime.

Right now, I manually ssh into the server, pull fresh code from my repo, re-run `scripts/run-prod.sh`, and pray to the gods that everything goes smoothly, which often enough it doesn't, for mysterious reasons, and in the meantime the app is down. Clearly, I'm doing it wrong. How is everyone else going about this?

**Comment by eXon** [comment]
Hey @skosch 

You should be adapting the scripts to publish the way you usually do it. All you need is to copy the build.sh script into deploy.sh. Then, change `meteor build ..` by the command you would normally use while you are in the meteor project folder.

You can use `mup` or anything else you were using with a regular meteor project.

By the way, this will be much simplier very soon because the webpack build will be integrated within the meteor build process. More to come soon :)

**Comment by skosch** [comment]
Thanks Benoît, that's encouraging, I'll give that a try. It's good to hear that a tighter integration is on the agenda. Happy Thanksgiving!
